### PR TITLE
Enable offline inbox for release builds.

### PR DESCRIPTION
**Pull request by mandrigin**
Also, reorder `.env.prod` so the keys order is the same as in the plain `.env`

fixes release build misconfiguration

status: ready

### CODE REVIEW

File: .env.prod [hunk 1]
```diff
@@ -2,22 +2,22 @@ TESTFAIRY_ENABLED=0
...
+MIXPANEL_TOKEN=2584e00100d319d12e538cc4d0fa9fc1
 DEFAULT_NETWORK=testnet_rpc
+TESTFAIRY_TOKEN=969f6c921cb435cea1d41d1ea3f5b247d6026d55
 INSTABUG_TOKEN=758630ed52864cbad9c5eeeac596c60c
 DEBUG_WEBVIEW=0
 INSTABUG_SURVEYS=1
 GROUP_CHATS_ENABLED=0
 FORCE_SENT_RECEIVED_TRACKING=0
 USE_SYM_KEY=0
+JSC_ENABLED=1
```
Comment (oskarth) [inline .env.prod]: I thought we removed JSC_ENABLED flag?

**Review by oskarth — APPROVED** [review]

File: .env.prod (hunk 1 above)
Comment (mandrigin) [reply .env.prod]: no idea, I just moved it down :)

Comment (mandrigin) [reply .env.prod]: (so when you look at `.env` and `.env.prod` side-by-side, it is useful :)

Comment (flexsurfer) [reply .env.prod]: https://github.com/status-im/status-react/pull/4051/files

Comment (flexsurfer) [reply .env.prod]: for some reason it was removed only in .env

**Review by flexsurfer — APPROVED** [review]

File: .env.prod (hunk 1 above)
Comment (mandrigin) [reply .env.prod]: cool, I'll remove it then

**Comment by mandrigin** [comment]
merging w/o manual QA since this is just a release config change

**Pull request merged**